From 88f9bd0136c786b499a171e196275fc2ba853f3b Mon Sep 17 00:00:00 2001
From: Mark Wiebe <399551+mwiebe@users.noreply.github.com>
Date: Mon, 4 Nov 2024 16:26:56 -0800
Subject: [PATCH] patched

---
 pyproject.toml | 21 ---------------------
 1 file changed, 21 deletions(-)

diff --git a/pyproject.toml b/pyproject.toml
index dc22099..bb3f888 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -49,27 +49,6 @@ artifacts = [
   "*_version.py",
 ]
 
-[tool.hatch.build.targets.wheel.force-include]
-"./maya_submitter_plugin" = "deadline/maya_submitter/maya_submitter_plugin"
-
-[tool.hatch.version]
-source = "vcs"
-
-[tool.hatch.version.raw-options]
-version_scheme = "post-release"
-
-[tool.hatch.build.hooks.vcs]
-version-file = "_version.py"
-
-[tool.hatch.build.hooks.custom]
-path = "hatch_custom_hook.py"
-
-[tool.hatch.build.hooks.custom.copy_version_py]
-destinations = [
-  "src/deadline/maya_adaptor",
-  "src/deadline/maya_submitter",
-]
-
 [tool.hatch.build.targets.sdist]
 include = [
     "src/*",
-- 
2.46.2.windows.1

